alb.tf

```diff
@@ -0,0 +1,72 @@
+resource "aws_lb" "alb" {
+  name               = "alb"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.http.id]
+  subnets            = [aws_subnet.public_1a.id, aws_subnet.public_1b.id]
+
+  enable_deletion_protection = false
+}
+
+resource "aws_lb_listener" "webserver" {
+  load_balancer_arn = aws_lb.alb.arn
+  port              = "80"
+  protocol          = "HTTP"
+
+  default_action {
+    type             = "forward"
+    target_group_arn = aws_lb_target_group.webserver.arn
+  }
+}
+
+resource "aws_lb_listener_rule" "phpmyadmin" {
+  listener_arn = aws_lb_listener.webserver.arn
+  priority     = 100
```

Comment:
it's better to keep priority default, in this case it will be incremented automaticaly, in case of setting a custome value it's possible to get a conflict by using multiple rules